src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+
+    private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        result.append("Report for period ")
+                .append(dateFrom).append(" - ").append(dateTo);
+        try {
+            int[] earned = new int[names.length];
+            Date fromDate = DATE_FORMAT.parse(dateFrom);
+            Date toDate = DATE_FORMAT.parse(dateTo);
```

Comment:
![image](https://github.com/mate-academy/jv-salary-info/assets/49767049/4bddb670-5e74-41e0-bd0c-a7cd9ae7527d)

Reply:
Oh...
I'm sorry, haven't even realized that the checklist is unique for each task.
Thank you for your patience!